org-code-javabuilder/lib/src/main/java/org/code/javabuilder/AWSOutputAdapter.java

```diff
@@ -63,13 +38,10 @@ public void sendDebuggingMessage(ClientMessage message) {
   }
 
   private void sendMessageHelper(PostToConnectionRequest post) {
-    if (!this.hasActiveConnection) {
-      return;
-    }
     try {
       this.api.postToConnection(post);
     } catch (GoneException e) {
-      this.hasActiveConnection = false;
+      throw new InternalServerRuntimeError(InternalErrorKey.CONNECTION_TERMINATED, e);
```

Comment:
General question -- how does this PR affect our current reporting of "SEVERE" errors? Will this be logged as severe?

Reply:
This particular error will not because it is caught in CodeBuilderRunnable and logged as a warning, but in general our SEVERE error reporting should not change - we should be reporting the same way for all internal exceptions.